test/TorchSharpTest/TestTorchVisionUtils.cs

```diff
@@ -0,0 +1,20 @@
+using System.IO;
+using Xunit;
+
+namespace TorchSharp
+{
+    public class TestTorchVisionUtils
+    {
+        [Fact]
+        public void Save_Image_TestMemoryUsage()
+        {
+            var imager = new torchvision.io.SkiaImager();
```

Comment:
@NiklasGustafsson Is there a mock for Imager, or is it ok to use SkiaImager here?

Reply:
No mock framework like NSubstitite or Moq?. It would be quite easy to add a Mock implementation of the interface, but I'll leave that for another time.

Reply:
No, not Moq. It's overkill here, I think.

https://www.bleepingcomputer.com/news/security/popular-open-source-project-moq-criticized-for-quietly-collecting-data/

Reply:
Yeah, 'we' heard! NSubstitute seems like a good alternative.